frontend/app/src/components/help/HelpSidebar.vue

```diff
@@ -42,26 +42,47 @@
           <v-icon>mdi-chevron-right</v-icon>
         </v-list-item-action>
       </v-list-item>
-      <v-list-item v-if="!$interop.isPackaged" selectable @click="openAbout()">
-        <v-list-item-avatar>
-          <v-icon>mdi-information</v-icon>
-        </v-list-item-avatar>
-        <v-list-item-content>
-          <v-list-item-title>{{ $t('help_sidebar.about') }}</v-list-item-title>
-          <v-list-item-subtitle>
-            {{ $t('help_sidebar.about_subtitle') }}
-          </v-list-item-subtitle>
-        </v-list-item-content>
-        <v-list-item-action>
-          <v-icon>mdi-chevron-right</v-icon>
-        </v-list-item-action>
-      </v-list-item>
+      <template v-if="!$interop.isPackaged">
+        <v-list-item selectable @click="openAbout()">
+          <v-list-item-avatar>
+            <v-icon>mdi-information</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              {{ $t('help_sidebar.about.title') }}
+            </v-list-item-title>
+            <v-list-item-subtitle>
+              {{ $t('help_sidebar.about.subtitle') }}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+          <v-list-item-action>
+            <v-icon>mdi-chevron-right</v-icon>
+          </v-list-item-action>
+        </v-list-item>
+        <v-list-item selectable @click="downloadBrowserLog()">
```

Comment:
Can we disable it if there are no logs in the indexeddb?

Reply:
I think we can keep the button enabled. And notify the user if they click download but there are no data logs. The idea is to makes the user familiar and aware with the existence of the button. Also I think it is kinda hard to make it reactive.

Reply:
Yes that works too, as long as an empty file is download doesn't start